client-cli/src/command.rs

```diff
@@ -97,6 +100,38 @@ pub enum Command {
     },
 }
 
+/// normal
+#[cfg(not(feature = "mock-enc-dec"))]
+fn get_tx_query(tendermint_client: WebsocketRpcClient) -> Result<DefaultTransactionObfuscation> {
+    let result = tendermint_client.query("txquery", &[])?.bytes()?;
+    let address = std::str::from_utf8(&result).chain(|| {
+        (
+            ErrorKind::ConnectionError,
+            "Unable to decode txquery address",
+        )
+    })?;
+    if let Some(hostname) = address.split(':').next() {
+        Ok(DefaultTransactionObfuscation::new(
+            address.to_string(),
+            hostname.to_string(),
+        ))
+    } else {
+        Err(client_common::Error::new(
+            ErrorKind::ConnectionError,
+            "Unable to decode txquery address",
+        ))
+    }
+}
+
+/// temporary
+#[cfg(feature = "mock-enc-dec")]
+fn get_tx_query(
+    tendermint_client: WebsocketRpcClient,
+) -> Result<MockAbciTransactionObfuscation<WebsocketRpcClient>> {
+    println!("WARNING: Using mock (non-enclave) infrastructure");
```

Comment:
Can also use `warn!` to give a better visual alert.

Reply:
client-cli doesn't use the env_logger -- I'll open an issue for that